/private/var/tmp/sage-10.0-current/local/var/lib/sage/venv-python3.11.1/bin/sage-preparse: File 'problem.sage' is missing
